src/js/api.ts

```diff
@@ -192,8 +192,9 @@ export class PyodideAPI {
       }
     }
     if (packages.size) {
-      await loadPackage(Array.from(packages), options);
+      return loadPackage(Array.from(packages), options);
```

Comment:
I think we would usually put an await here. It doesn't really change the behavior but perhaps it's a bit clearer.
```suggestion
      return await loadPackage(Array.from(packages), options);
```